src/components/Votes/Votes.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import { updateVotes } from "../../utils/api";
+import * as BiIcons from "react-icons/bi";
+import "./Votes.css";
+
+const Votes = ({ review_id, votes }) => {
+  const [voteInc, setVoteInc] = useState(0);
+
+  const handleClick = (click) => {
+    setVoteInc((curVotes) => {
+      if (1 === click) {
+        return curVotes + 1;
+      } else {
+        return curVotes - 1;
+      }
+    });
+    updateVotes(review_id, click).catch((err) => {
+      setVoteInc((currVotes) => {
+        return currVotes + 0;
+      });
+    });
+  };
```

Comment:
Users should only be able to vote once in either direction, at the moment they can upvote or downvote as much as they like - some state to track whether a button has been clicked to disable it conditionally might help